src/patternfly/components/Menu/menu.scss

```diff
@@ -0,0 +1,294 @@
+
+  // description
+  --pf-c-menu__item-description--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-menu__item-description--Color: var(--pf-global--Color--200);
+
+  // icon
+  --pf-c-menu__item-icon--MarginRight: var(--pf-global--spacer--sm);
+
+  // toggle icon
+  --pf-c-menu__item-toggle-icon--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-toggle-icon--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // select icon
+  --pf-c-menu__item-select-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-select-icon--Color: var(--pf-global--active-color--100);
+  --pf-c-menu__item-select-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // external icon
+  --pf-c-menu__item-external-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-external-icon--Color: var(--pf-global--link--Color);
+  --pf-c-menu__item-external-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // action
+  --pf-c-menu__item-action--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action-icon--Color: var(--pf-global--Color--dark-200);
+  --pf-c-menu__item-action-icon--Height: calc(var(--pf-c-menu__item--FontSize) * var(--pf-c-menu__item--LineHeight));
+  --pf-c-menu__item-action--hover__icon--Color: var(--pf-global--Color--dark-100);
+  --pf-c-menu__item-action--m-favorite__icon--Color: var(--pf-global--disabled-color--200);
+  --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
+
+  padding-top: var(--pf-c-menu--PaddingTop);
+  padding-bottom: var(--pf-c-menu--PaddingBottom);
+  background-color: var(--pf-c-menu--BackgroundColor);
+  box-shadow: var(--pf-c-menu--BoxShadow);
+
+  &.pf-m-flyout {
+    .pf-c-menu__sublist {
+      position: absolute;
+      top: var(--pf-c-menu--m-flyout__sublist--Top);
+      left: 100%;
+    }
+
+    .pf-c-menu__list-item {
+      &:hover,
+      &:focus-within {
+        color: unset;
+        background-color: unset;
+      }
+    }
+
+    .pf-c-menu__item {
+      &:hover,
+      &:focus-within {
+        color: var(--pf-c-menu__list-item--hover--Color);
+        background-color: var(--pf-c-menu__list-item--hover--BackgroundColor);
+      }
+    }
+  }
+
+  .pf-c-divider {
+    margin-top: var(--pf-c-menu--c-divider--MarginTop);
+    margin-bottom: var(--pf-c-menu--c-divider--MarginBottom);
+  }
+}
+
+// Header
+.pf-c-menu__header {
+  padding-top: var(--pf-c-menu__header--PaddingTop);
+  padding-right: var(--pf-c-menu__header--PaddingRight);
+  padding-bottom: var(--pf-c-menu__header--PaddingBottom);
+  padding-left: var(--pf-c-menu__header--PaddingLeft);
+
+  .pf-c-menu__item {
```

Comment:
do you have an example with an item in a header? Also assuming that's needed, could you just move the `__item--Padding` vars up under `__header`?

Reply:
Yes the example was in the drill-down example where I nested the item inside of the header, I removed the padding from the item because the header has the padding, so i cant imagine that the item would need to have padding, what do you think? I guess we could also have a new container for the drill down in the header ? @mattnolting what do you think 

<img width="1152" alt="Screen Shot 2020-08-20 at 11 14 50 AM" src="https://user-images.githubusercontent.com/20118816/90790927-9f32b280-e2d6-11ea-8aa4-e45b8642c571.png">